README.md

```diff
@@ -1,113 +1,79 @@
-$ kubectl create namespace istio-system
-$ helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr
-```
+1. A cluster _without_ Istio already installed
+2. cert-manager [installed](https://cert-manager.io/docs/installation/) in the cluster
+3. An `Issuer` or `ClusterIssuer` which will be used to issue Istio certificates
+4. istio-csr installed (likely via helm)
+5. Istio [installed](https://istio.io/latest/docs/setup/install/istioctl/) with some required custom config, e.g. in `./hack/istio-config-1.10.0.yaml`
 
-#### Load root CAs from file ca.pem (Preferred)
+### Why Custom Istio Install Manifests?
 
-We use a Secret volume here to source the root CAs, however projects such as
-[secrets-store-csi-driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver)
-are great options for improving security further.
+If you take a look at the contents of [`istio-config-1.10.0.yaml`](./hack/istio-config-1.10.0.yaml) there are a few
+custom configuration options which are important.
 
-```terminal
-$ helm repo add jetstack https://charts.jetstack.io
-$ helm repo update
-$ kubectl create namespace istio-system
-$ kubectl create secret generic istio-root-ca --from-file=ca.pem=ca.pem -n cert-manager
-$ helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr \
-  --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
-  --set "volumeMounts[0].name=root-ca" \
-  --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
-  --set "volumes[0].name=root-ca" \
-  --set "volumes[0].secret.secretName=istio-root-ca"
-```
+Requried changes include setting `ENABLE_CA_SERVER` to `false` and setting the `caAddress` from which Istio will
+request certificates; replacing the CA server is the whole point of istio-csr!
 
-All helm value options can be found [here](./deploy/charts/istio-csr/README.md).
+Mounting and statically specifying the root CA is also an important recommended step. Without a manually specified
+root CA istio-csr defaults to trying to discover root CAs automatically, which could theoretically lead to a
+[signer hijacking attack](https://github.com/cert-manager/istio-csr/issues/103#issuecomment-923882792).
 
-### Installing Istio
+### Issuer or ClusterIssuer?
 
-If you are running Openshift, prepare the cluster for Istio.
-Follow instructions from Istio [platform setup
-guide](https://istio.io/latest/docs/setup/platform-setup/openshift/).
+We'd recommend an `Issuer` unless you've got some reason to use a `ClusterIssuer`.
 
-Finally, install Istio. Istio must be installed using the IstioOperator
-configuration changes within
-[`./hack/istio-config-x.yaml`](./hack/istio-config-1.10.0.yaml). Later versions
-of Istio share the same config.
+It's much easier to reason about the access controls for an `Issuer`. In addition, `Issuer`s will ultimately require less work
+to configure, since our examples in the [getting started](./docs/getting_started.md) guide focus on `Issuer`s.
```

Comment:
Maybe we can add back the warning about the fact that the ACME issuer with Let's Encrypt is not supported in istio-csr due to the fact that the SPIFFE ID cannot be added to the X.509 subjectAltName field.

Reply:
Done, good idea again! :+1: